defaults.env

```diff
@@ -1,5 +1,6 @@
 export CORECLR_ENABLE_PROFILING=1
 export CORECLR_PROFILER='{918728DD-259F-4A6A-AC2B-B85E1B658318}'
 export CORECLR_PROFILER_PATH=/opt/opentelemetry-dotnet-autoinstrumentation/OpenTelemetry.ClrProfiler.Native.so
+export DOTNET_STARTUP_HOOKS=/opt/opentelemetry-dotnet-autoinstrumentation/OpenTelemetry.Instrumentation.StartupHook.dll
```

Comment:
That's a good question. I'm wondering if the `Instrumentation` part of the name is something that we should avoid since that's a pattern used for the actual instrumentation libraries.

Reply:
related: we should consider the path too. While it is just the "defaults.env" a shorter name seems better, eg.: `/opt/opentelemetry/dotnet/<assembly_dll>`. It requires changing the path at other locations... likely it is better handled when we decide about the rename per SIG meeting conversation.